CIPs/CIP-85/CIP-85.md

```diff
@@ -50,116 +50,121 @@ A Collection "instance" would therefore be made of 1 `AppendCollection` document
 
 The `AppendCollection` schema must be an `object` with a `$ceramic` field from [CIP-88](../CIP-88/CIP-88.md) having the type `appendCollection` and pointing to the slice's `schema`, along with the following properties:
 
-- `sliceSize`: the maximum number of items a single slice should contain
+- `sliceMaxItems`: the maximum number of items a single slice should contain
 - `slicesCount`: the total number of slices the collection contains
-- `first`: DocID of the first slice
-- `last`: DocID of the last slice
 
 ```js
 {
   $schema: 'http://json-schema.org/draft-07/schema#',
-  $ceramic: { type: 'appendCollection', schema: '<slice schema ID>' },
+  $ceramic: { type: 'appendCollection', sliceSchema: '<slice schema ID>' },
   title: 'MyCollection',
   type: 'object',
   properties: {
-    sliceSize: { type: 'integer', minimum: 10, maximum: 256 },
+    sliceMaxItems: { type: 'integer', minimum: 10, maximum: 256 },
     slicesCount: { type: 'integer', minimum: 1 },
```

Comment:
```suggestion
    slicesCount: { type: 'integer', minimum: 0 },
```

Reply:
Needed to create the initial empty AppendCollection to get the DocID

Reply:
I'm wondering if really it's needed?
Setting it to `0` to get started means creating the collection, then the slice, and finally updating the collection, but if we set the `slicesCount` to `1` when creating the collection, we can then create the slice and no need to update the collection.

Reply:
Okay yeah good point.  Worst case scenario when you load the index 0 slice it has no content yet - no biggie.